reference/faq.md

```diff
@@ -1,16 +1,26 @@
 
-Pulumi uses pulumi.com to store information about the current state of your application, which is used during updates, previews and destroys as the source of truth for the current state of your cloud resources. We refer to this state as the "checkpoint" for your application. In addition, pulumi.com ensures that for a given stack, only a single update is running at once (so, if you and someone else are collaborating on a stack together, pulumi.com ensures that you both don't update the same stack at the same time). Once your stack has been deployed, it has no dependency on pulumi.com. To learn more about how the Pulumi engine uses pulumi.com, see [How Pulumi Works](https://pulumi.io/reference/how.html).
+Pulumi uses pulumi.com to store information about the current state of your application, which is used during updates, previews and destroys as the source of truth for the current state of your cloud resources. We refer to this state as the "checkpoint" for your application. In addition, pulumi.com ensures that for a given stack, only a single update is running at once (so, if you and someone else are collaborating on a stack together, pulumi.com ensures that you both don't update the same stack at the same time.) Once your stack has been deployed, it has no dependency on pulumi.com. To learn more about how the Pulumi engine uses pulumi.com, see [How Pulumi Works](https://pulumi.io/reference/how.html).
 
 ## What happens if pulumi.com is down?
 
-If pulumi.com is down, you'll be unable to preview, update or destroy a stack. But, any infrastructure that you’ve deployed through Pulumi will continue working and can be managed with your cloud provider’s console or CLI. 
+Any infrastructure that you’ve deployed using Pulumi will continue working and can be managed with your cloud provider’s console or CLI, that is, pulumi.com should not effect any runtime behavior of your application.  
 
-Some commands, like `pulumi logs`, use pulumi.com to find the correct log stream. This command will not work if pulumi.com is down, but your cloud provider will still produce logs that you can view through a cloud console or CLI. 
+If pulumi.com is down, you'll be unable to preview, update or destroy a stack using Pulumi.  Some commands, like `pulumi logs`, use pulumi.com to find the correct log stream, so will function until pulumi.com recovers, however, your cloud provider will still produce logs that you can use for diagnostics and you can view these via your cloud console or CLI. 
+
+## How can I add support for my favorite cloud?
+
+To enable a new cloud, you need to create a Pulumi Resource Provider.  This requires a gRPC interface, and can be implemented directly; see see https://github.com/pulumi/pulumi-kubernetes for an example of this.  
+
+If there is an existing Terraform Resource Provider for the target, you can also use Terraform Bridge, for an example of this approach, see https://github.com/pulumi/pulumi-aws/blob/master/resources.go.
```

Comment:
Probably worth pointing to https://github.com/pulumi/pulumi-terraform/blob/master/README.md also, since it describes the overall structure and process of adding a new provider using the bridge.